src/java.base/share/classes/java/io/BufferedInputStream.java

```diff
@@ -50,6 +50,12 @@
  * reread before new bytes are  taken from
  * the contained input stream.
  *
+ * <p> More than one instance of {@code BufferedInputStream} should not be
+ * used with the same underlying {@code InputStream} instance.  Doing
```

Comment:
Nit - there's one extra space here before the `Doing`.

Reply:
Fixed in d6831f9.